packages/react-components/lib/tasks/create-task.tsx

```diff
@@ -873,64 +898,67 @@ export function CreateTaskForm({
                       })}
                     </TextField>
                   </Grid>
-                  <Grid item xs={isScreenHeightLessThan800 ? 6 : 7}>
-                    <DateTimePicker
-                      value={new Date()}
-                      onChange={() => 0}
-                      label="Start Time"
-                      disabled
-                    />
-                  </Grid>
                   <Grid item xs={1}>
-                    <Checkbox
-                      checked={warnTime !== null}
-                      onChange={handleWarnTimeCheckboxChange}
-                      inputProps={{ 'aria-label': 'controlled' }}
-                      sx={{
-                        '& .MuiSvgIcon-root': { fontSize: isScreenHeightLessThan800 ? 22 : 32 },
-                      }}
-                    />
+                    <Tooltip title="Ongoing tasks where estimated finish times are past their specified warning times will be flagged.">
+                      <Switch checked={warnTime !== null} onChange={handleWarnTimeSwitchChange} />
+                    </Tooltip>
                   </Grid>
-                  <Grid item xs={isScreenHeightLessThan800 ? 5 : 4}>
+                  <Grid item xs={8}>
                     <DateTimePicker
                       disabled={warnTime === null}
                       value={warnTime}
                       onChange={(date) => {
                         setWarnTime(date);
                       }}
                       label="Warn Time"
+                      sx={{ marginLeft: theme.spacing(1) }}
                     />
                   </Grid>
+                  <Grid container xs={3} justifyContent="flex-end" alignItems="center">
+                    <Tooltip title="Prioritized tasks will added to the front of the task execution queue">
+                      <Chip
+                        icon={<BoltIcon />}
```

Comment:
1. Not sure about a bolt icon, to me a bolt implies it is faster which is not true for high priority tasks, material does have a "low priority" icon https://fonts.google.com/icons?icon.query=prior, we can probably flip it into a high priority with some css magic, imo that would be more intuitive.
1. The priority should be a checkbox.

Reply:
A checkbox usually implies multiple choices/checkboxes, that a user can pick and customize, so I'd rather use a toggle switch (like warning time now)

https://github.com/open-rmf/rmf-web/pull/1000/commits/0703b77068deaf20957ffb72fe6a598c28a2c0d3